html_css/intermediate_html/emmet.md

```diff
@@ -1,51 +1,49 @@
 ### Introduction
 
-Emmet is a plugin, built into VSCode, that helps you write HTML and CSS more efficiently by providing a bunch of clever shortcuts. By now, you have written a good amount of HTML and CSS, and if you have been using vscode there's a big chance that you have already encountered emmet in some way.
+Emmet is a plugin, built into VS Code, that helps you write HTML and CSS more efficiently by providing a bunch of clever shortcuts. By now, you have written a good amount of HTML and CSS, and if you have been using VS Code there's a big chance that you have already encountered Emmet in some way.
 
 ### Learning Outcomes
 
 By the end of this lesson, you should be able to do the following:
 
-- Use some of emmets most useful shortcuts.
-- Set up custom emmet keybindings in vscode.
+- Use some of Emmet's most useful shortcuts.
+- Set up custom Emmet keybindings in VS Ccode.
 
 ### Emmet
 
-Emmet is a really useful tool for anyone that works with HTML and CSS a lot and luckily does not have that big of a learning curve because if you already know how to write HTML and CSS you'll have no trouble picking up emmet abbreviations.
+Emmet is a really useful tool for anyone that works a lot with HTML and CSS. Luckily Emmet does not have that big of a learning curve, and if you already know how to write HTML and CSS you'll have no trouble picking up Emmet abbreviations.
```

Comment:
I'm going to be annoying here and add a comma haha. I love taking breaks when talking!

```suggestion
Emmet is a really useful tool for anyone that works a lot with HTML and CSS. Luckily, Emmet does not have that big of a learning curve, and if you already know how to write HTML and CSS you'll have no trouble picking up Emmet abbreviations.
```